pippi

```diff
@@ -43,11 +43,15 @@ def main(arguments):
       try:
         command(arguments[2:])
       except BaseException as err:
-        print()
-        print('Running pippi failed in '+command.__name__+' operation, due to error:')
-        print(err)
-        print()
-        sys.exit()
+        clean_exit = False
+        if isinstance(err, SystemExit):
+          clean_exit = True if err.code == 0 else clean_exit
```

Comment:
I find this construction kind of convoluted and therefore a bit difficult to follow.  How about something like just
```python
clean_exit = isinstance(err, SystemExit) and err.code == 0
```
?